src/css/main.less

```diff
@@ -680,22 +677,22 @@ fieldset[disabled] .btn-info.active {
     .icon {
       font-size: 1.4em;
     }
-    padding: 3px 12px;
+    padding: (@base-font-size * 0.5) (@base-spacing-unit * 2);
```

Comment:
With syntax highlighting it's a little bit better readable:
```css
padding: @base-font-size * 0.5 @base-spacing-unit * 2;
```
The parenthesis distracts me and makes the things inconsistent, if you are only using one value:
```css
padding: (@base-font-size * 0.5);
```
No-Go.